qml/Secrets/main.cpp

```diff
@@ -38,6 +38,7 @@ void Sailfish::Secrets::Plugin::SecretsPlugin::registerTypes(const char *uri)
 
     qmlRegisterUncreatableType<Sailfish::Secrets::Request>(uri, 1, 0, "SecretsRequest", QStringLiteral("Request is an abstract class, can't construct in QML"));
     qRegisterMetaType<Sailfish::Secrets::Request::Status>("SecretsRequestStatus");
+    qmlRegisterUncreatableType<Sailfish::Secrets::PluginInfo>(uri, 1, 0, "PluginInfo", QStringLiteral("PluginInfo objects cannot be constructed directly in QML"));
```

Comment:
To avoid name collision:

```
qmlRegisterUncreatableType<Sailfish::Secrets::PluginInfo>(uri, 1, 0, "SecretsPluginInfo", QStringLiteral("PluginInfo objects cannot be constructed directly in QML"));
```

Reply:
Again, what's the reason for this?